### ISSUE TITLE
evaluation code question

### ISSUE THREAD

**Issue by kvelleng** [cvjena/GroVE-Probabilistic_VLM_embeddings#1]
Hi!

Thanks a lot for sharing your code, started to go through it and I was curious if the following was on purpose in the evaluation.py file:
line 81 -->  learned_txt_embeddings_variance.append(Z_img_variance.detach())
Shouldn't this be Z_txt_variance instead?

Cheers!

**Comment by vaishwarya96** [comment]
Hey,

Thanks a lot for catching that! That was a typo on my part. I've corrected it in the latest update.
Appreciate you taking the time to go through the code!

Best regards,
Aishwarya
